### PR TITLE
Move rpmrepo to AWS

### PR DESCRIPTION
**Pull request by ambv**
- upgrade to Centos 8
- storage in S3
- host key, authorized_keys, and GPG keys in AWS SecretsManager
- inoticoming instead of incron
- gosu instead of sudo
- visor.py instead of bash-based daemon supervising

### CODE REVIEW

File: server/containers/rpmrepo/Dockerfile [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 ENV REPO_INCOMING_DIR=/var/spool/repo/incoming/
 
 RUN yum install -y \
-        https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm \
+        https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm \
     && yum install -y \
-        createrepo openssh-server bash incron sudo gnupg rpm-sign rsync \
-        yum-utils gcc python3 python3-devel libffi-devel \
-    && pip3 install gsutil \
+        createrepo openssh-server bash gnupg rpm-sign wget \
```
Comment (ambv) [inline server/containers/rpmrepo/Dockerfile:9]: I used wget because curl didn't follow GitHub redirects and downloaded the HTML redirect body instead.

Comment (elprans) [reply server/containers/rpmrepo/Dockerfile:9]: `curl -L`?

**Review by elprans — APPROVED** [review]
I wonder if we can avoid duplicating the common scripts like `fetch_secrets.py` and `visor`

**Comment by ambv** [comment]
1. Either by publishing those things on PyPI (this repo is open anyway); or
2. By publishing those tools on Docker Hub (FROM scratch) and then copying them here in a layer.

A variant of 1. could be to just put them on GitHub and download them manually in the Dockerfile, just like we're downloading inoticoming right now, just without the build step.

**Pull request merged**